core/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "mc-core"
+version = "2.0.0"
+authors = ["MobileCoin"]
+edition = "2021"
+description = "MobileCoin Core Library"
+readme = "README.md"
+
+[features]
+bip39 = ["tiny-bip39", "slip10_ed25519"]
```

Comment:
optional: consider using `dep:tiny-bip39` and `dep:slip10_ed25519` to emphasize that those are dependencies and not features. I understand that you're using `dep:serde` out of neccesity to disambiguate with the feature also called `serde`, but I also find `dep:` useful in that it distinguishes dependencies from features.

https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies